deckhouse-controller/pkg/controller/module-controllers/downloader/module_downloader.go

```diff
@@ -137,6 +138,24 @@ func (md *ModuleDownloader) DownloadMetadataFromReleaseChannel(moduleName, relea
 	return res, nil
 }
 
+func (md *ModuleDownloader) FetchModuleVersionFromReleaseChannel(moduleName string, releaseChannel string) (moduleVersion string, err error) {
```

Comment:
NIT: we try not to use named returned values. 
The better way is to explicitly return them like `return moduleVersion, err`

Reply:
This method is no longer used, I have removed it